package.json

```diff
@@ -21,18 +21,26 @@
   "bugs": {
     "url": "https://github.com/codealchemist/youtube-audio-server/issues"
   },
+  "files": ["!/temp-audio/**", "src/*", "!/src/test"],
   "homepage": "https://github.com/codealchemist/youtube-audio-server#readme",
   "dependencies": {
     "express": "^4.17.1",
     "express-no-favicons": "0.0.1",
+    "ffmetadata": "^1.6.0",
     "fluent-ffmpeg": "^2.1.2",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
+    "qs": "^6.9.1",
```

Comment:
Newer versions of `ytdl-core` already support passing a video ID or URL so we don't need to parse the URL now, yay!